plasmapy/formulary/tests/test_frequencies.py

```diff
@@ -32,6 +33,9 @@
         (wc_, gyrofrequency),
         (wlh_, lower_hybrid_frequency),
         (wuh_, upper_hybrid_frequency),
+        (wbb_, Buchsbaum_frequency),
+        (wii_, Buchsbaum_frequency),
+        (wbi_, Buchsbaum_frequency),
```

Comment:
```suggestion
```
It looks like the aliases were deleted, in which case we can remove the tests too.

Reply:
Indeed.